Description: Set Background for BL7
 Use BodhiLeaf-Cjn as thw default bckground/
Author:  Robert (ylee) Wiley <ylee@bodhilinux.com>
Origin: other
Forwarded: no
Last-Update: 2023-02-12
--- a/config/bodhi/e.src
+++ b/config/bodhi/e.src
@@ -1932,6 +1932,7 @@ group "E_Config" struct {
     value "config_version" int: 1000019;
     value "show_splash" int: 1;
     value "init_default_theme" string: "/usr/share/enlightenment/data/themes/MokshaGreen.edj";
+    value "desktop_default_background" string: "/usr/share/enlightenment/data/backgrounds/BodhiLeaf-Cjn.edj";
     value "desktop_default_name" string: "%i-%i";
     value "desktop_default_window_profile" string: "standard";
     value "menus_scroll_speed" double: 1000.0;

